src/components/internal/DataViewFilterMultiselectDropdownContent.jsx

```diff
@@ -375,7 +375,7 @@ function DataViewFilterMultiselectDropdownContent(props) {
 					{
 						showList &&
 						<CheckboxList
-							checked={state.selected}
+							checked={optionsDisabled ? [] : state.selected}

```

Comment:
Although this solution solves the issue visually, internally the dataview still receives the options selected, which shouldn't happen. Here you can see that even though the selected filter is "exist" it still gives me the ids of the 2 first options: 
![image](https://user-images.githubusercontent.com/23409284/181384965-2d6d8a58-f4b3-4d2e-bf7c-278f5f3576a8.png)
What I suggest doing here would be updating state.selected to be [] and keeping the previously selected options on another state variable, that way you can retrieve them if you select another comparison back again. I believe this can be solved on this same file but inside the "menuItems" variable.